src/pages/settings/Report/ReportSettingsPage.js

```diff
@@ -80,7 +81,7 @@ class ReportSettingsPage extends Component {
 
         return (
             <ScreenWrapper>
-                <FullPageNotFoundView shouldShow={_.isEmpty(this.props.report)}>
+                <FullPageNotFoundView shouldShow={shouldDisableSettings}>
```

Comment:
After the change FullPageNotFoundView was not checking whether this.props.report is empty. I'll add it for clear message that we don't wont to show screen when no report is found

Reply:
Thank you